api/models/User.js

```diff
@@ -2,25 +2,34 @@
 
 const mongoose = require('mongoose');
 
+var User;
+
 // User Schema Definition
 var userSchema = new mongoose.Schema({
-	name: String
+	name: {type: String , required: true},
+	isAdmin: {type: Boolean , default: false},
+	isMentor: {type: Boolean , default: false},
+	skills: [String],
+	googleID: {type: String , unique: true, required: true}
 });
 
 // User Schema Methods
 
 /**
  * Creates and saves a User object in the database
- * @param {string} user's name
+ * @param {string} user's name, {string} user's GoogleID (as hash)
  * @returns {User} newly saved User object
  * @example
- * User.create("Joe Bruin")
+ * User.create('Joe Bruin', 'someGoogleIdHash')
  *     .then(user => console.log(user))
  *     .catch(error => console.error(error));
  */
-userSchema.statics.create = function(name) {
+
+userSchema.statics.create = function(name, googleId) {
 	var user = new this({
-		name: name
+		name: name,
+		skills: [],
+		googleID: googleId
```

Comment:
The different naming conventions here are kinda inconsistent. Let’s keep it as googleId on the schema.